docs/application/dotnet/guides/applications/nui-widget-app.md

```diff
@@ -0,0 +1,244 @@
+
+    - `OnTerminate()` callback is triggered when the widget instance is terminated.
+
+      ```csharp
+      protected override void OnTerminate(string contentInfo, TerminationType type)
+      ```
+
+    - `OnPause()` callback is triggered when the widget instance is paused.
+
+      Your application does the necessary actions when the widget instance becomes invisible because the Application framework can destroy a paused widget instance, for instance, such as the case that memory is lacked.
+
+      ```csharp
+      protected override void OnPause()
+      ```
+
+    - `OnResume()` callback is triggered when the widget instance is resumed.
+
+      ```csharp
+      protected override void OnResume()
+      ```
+
+    - `OnResize()` callback is triggered when the widget instance is resized.
+
+      Your application does the necessary actions to accommodate the new size.
+
+      ```csharp
+      protected override void OnResize(Window window)
+      ```
+
+    - `OnUpdate()` callback is triggered when a widget update event is received.
+
+      Your application does the necessary actions for the widget update. If the `isForce` parameter is true, the widget can be updated even in the pause state.
+      Your application does the necessary actions when the widget instance becomes visible.
+
+      ```csharp
+      protected override void OnUpdate(string contentInfo, int isForce)
+      ```
+
+6. Drawing the Widget UI in `OnCreate()`:
+
+    Initialize resources for this widget instance and draw the content on the screen.
+
+    The widget UI is drawn in the `OnCreate()` callback of your widget class:
+
+    ```csharp
+    protected override void OnCreate(string contentInfo, Window window)
+    {
+        View rootView = new View();
+        rootView.BackgroundColor = Color.White;
+        rootView.Size2D = window.Size;
+        rootView.PivotPoint = PivotPoint.Center;
+        window.GetDefaultLayer().Add(rootView);
+
+        TextLabel sampleLabel = new TextLabel("Hello World!");
+        sampleLabel.FontFamily = "SamsungOneUI 500";
+        sampleLabel.PointSize = 71;
+        sampleLabel.TextColor = Color.Black;
+        sampleLabel.SizeWidth = 200;
+        sampleLabel.PivotPoint = PivotPoint.Center;
+
+        rootView.Add(sampleLabel);
+    }
+    ```
+
+## Create a NUI Widget View
+
+If necessary, you can create a Widget Viewer that shows UI drawn by Widget.
+
+To create a nui widget view:
+1. Add privileges for viewer to `tizen-manifest.xml`:
+
+    ```XML
+      <privileges>
+        <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+      </privileges>
+    ```
+2. Create widget manager and request add widget to widget manager.
+
+    You can create a widget manager using two different constructors:
+
+    - The `NUIWidgetApplication(Type type)` constructor for adding a widget using the widget app ID.
```

Comment:
Please add colon instead of period as a code follows:

```suggestion
- The `NUIWidgetApplication(Type type)` constructor for adding a widget using the widget app ID:
```